style.css

```diff
@@ -878,6 +878,209 @@ span {
   margin-bottom: 8px;
 }
 
+.desktop {
```

Comment:
- [ ] The popup menu is not displaying correctly on the mobile view, please consider fixing it

![Screenshot_20220519_060447](https://user-images.githubusercontent.com/25227022/169212228-de77e400-ab5f-4135-b113-eb60805522d6.png)